kubernetes/deployment.yml

```diff
@@ -0,0 +1,210 @@
+            - mountPath: /home/metro
+              name: metro-home-vol
+          envFrom:
+            - configMapRef:
+                name: metro-env
+        - command:
+          - /scripts/configure-geth.sh
+          name: configure-geth
+          image: ghcr.io/astriaorg/go-ethereum:0.0.3
+          volumeMounts:
+            - mountPath: /scripts/
+              name: executor-scripts-volume
+            - mountPath: /home/geth
+              name: executor-home-vol
+          envFrom:
+            - configMapRef:
+                name: geth-env
+        - command:
+          - /scripts/init-geth.sh
+          name: init-geth
+          image: ghcr.io/astriaorg/go-ethereum:0.0.3
+          volumeMounts:
+            - mountPath: /scripts/
+              name: executor-scripts-volume
+            - mountPath: /home/geth
+              name: executor-home-vol
+          envFrom:
+            - configMapRef:
+                name: geth-env
+        - command:
+          - /scripts/configure-metro.sh
+          name: configure-metro
+          image: ghcr.io/tomwright/dasel:alpine
+          envFrom:
+            - configMapRef:
+                name: metro-env
+          volumeMounts:
+            - mountPath: /scripts/
+              name: metro-scripts-volume
+            - mountPath: /home/metro
+              name: metro-home-vol
+      containers:
+        - name: celestia-app
+          command: ["/scripts/start-celestia-appd.sh"]
+          image: "ghcr.io/astriaorg/celestia-app:v0.11.0"
+          envFrom:
+            - configMapRef:
+                name: celestia-env
+          volumeMounts:
+            - mountPath: /scripts/
+              name: celestia-appd-scripts-volume
+            - mountPath: /home/celestia
+              name: celestia-home-vol
+          ports:
+            - containerPort: 26657
+        - name: celestia-bridge
+          command:
+          - /scripts/start-bridge.sh
+          image: "ghcr.io/astriaorg/celestia-node:test-sha-07fa3e7"
+          volumeMounts:
+            - mountPath: /scripts/
+              name: bridge-scripts-volume
+            - mountPath: /home/celestia
+              name: celestia-home-vol
+          envFrom:
+            - configMapRef:
+                name: celestia-env
+          ports:
+            - containerPort: 26659
+              name: bridge-svc
+        - name: metro-sequencer
+          command: ["/scripts/start-metro.sh"]
+          image: "ghcr.io/astriaorg/metro:0.0.3"
+          ports:
+            - containerPort: 1318
+              name: sequencer-svc
+            - containerPort: 9100
+          volumeMounts:
+            - mountPath: /scripts/
+              name: metro-scripts-volume
+            - mountPath: /home/metro
+              name: metro-home-vol
+          envFrom:
+            - configMapRef:
+                name: metro-env
+        - name: astria-sequencer-relayer
+          image: "ghcr.io/astriaorg/sequencer-relayer:0.0.4"
```

Comment:
where is this image/version from? since we moved to a monorepo, there aren't any tags, should this be updated to point to the crate in the monorepo?

Reply:
These packages still exist in our [gh image repository](https://github.com/orgs/astriaorg/packages) even though the old repo is archived, and there is a separate issue for me to implement image building in ci/cd for the new monorepo!

https://github.com/astriaorg/astria/issues/42